resource-managers/kubernetes/docker/src/main/dockerfiles/init-container/Dockerfile

```diff
@@ -0,0 +1,24 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM spark-base
+
+# If this docker file is being used in the context of building your images from a Spark distribution, the docker build
+# command should be invoked from the top level directory of the Spark distribution. E.g.:
+# docker build -t spark-init:latest -f dockerfiles/init-container/Dockerfile .
```

Comment:
`kubernetes/dockerfiles/..` instead of `dockerfiles/..`

Btw, only nits but seems like paths here in `Dockerfile`s for driver/executor are wrong: `kubernetes/dockerfiles/driver/Dockerfile` and `kubernetes/dockerfiles/executor/Dockerfile` respectively?

Reply:
Done.